teerex/src/lib.rs

```diff
@@ -91,8 +91,8 @@ decl_event!(
 		Forwarded(ShardIdentifier),
 		ShieldFunds(Vec<u8>),
 		UnshieldedFunds(AccountId),
-		CallConfirmed(AccountId, H256),
-		BlockConfirmed(AccountId, H256),
+		ParentchainBlockProcessed(AccountId, H256, H256),
+		SidechainBlockProposed(AccountId, H256),
```

Comment:
Here, the naming is actually ok, but I would change the word order to be in line with the suggested changes below, and with the other events.